tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -304,6 +363,20 @@ Object.PCM {
 			formats	'S16_LE,S24_LE,S32_LE'
 		}
 	}
+	pcm."98" {
+		name	"DMIC Raw 1"
+		id	98
+		direction	"capture"
+		Object.Base.fe_dai."DMIC Raw 1" {}
+
+		Object.PCM.pcm_caps."capture" {
+			name	"DMIC Raw 1"
+			# only 32-bit capture supported now
+			formats	'S32_LE'
```

Comment:
is S32_LE required for DMIC ?  We need to use S16_LE for AEC feature. If we have to use S32_LE, we need to add format converter

Reply:
> is S32_LE required for DMIC ? We need to use S16_LE for AEC feature. If we have to use S32_LE, we need to add format converter

@RanderWang, yes AEC feature needs S16_LE.
adding @cujomalainey